crates/miden-objects/src/errors.rs

```diff
@@ -324,8 +324,23 @@ pub enum AssetError {
       expected_ty = AccountType::NonFungibleFaucet
     )]
     NonFungibleFaucetIdTypeMismatch(AccountIdPrefix),
-    #[error("{0}")]
-    TokenSymbolError(String),
+    #[error("provided token symbol invalid")]
+    TokenSymbolError(#[source] TokenSymbolError),
```

Comment:
It seems like this is unused. Do we need it?

Reply:
My thought was that potentially we will want to return `TokenSymbolError` higher where we will use `AssetError` as a returning value. But probably it will be better to remove it for now and add again if such need will appear.